tools/pycafe/create_pycafe_links.py

```diff
@@ -0,0 +1,131 @@
+"""Generate PyCafe links for the example dashboards and post them as a comment on the pull request and as status."""
+
+import base64
+import datetime
+import gzip
+import json
+import os
+import subprocess
+import sys
+import textwrap
+from pathlib import Path
+from typing import Optional
+from urllib.parse import quote, urlencode
+
+from github import Auth, Github
+
+GITHUB_TOKEN = str(os.getenv("GITHUB_TOKEN"))
+REPO_NAME = str(os.getenv("GITHUB_REPOSITORY"))
+PR_NUMBER = int(os.getenv("PR_NUMBER"))
+RUN_ID = str(os.getenv("RUN_ID"))
+PACKAGE_VERSION = subprocess.check_output(["hatch", "version"]).decode("utf-8").strip()
```

Comment:
```suggestion
import vizro
PACKAGE_VERSION =  vizro.__version__
```

Can we get this direct from the source?

Reply:
In fact why do we even need this? Presumably there's only ever one wheel file in `dist` so we can just find the name of the file by doing `Path("dist").glob("*.whl")` and taking it from there, no need to manually build the filename again?